Index: solr-packaging/solr/core/src/java/org/apache/solr/core/SolrCore.java
===================================================================
--- solr-packaging.orig/solr/core/src/java/org/apache/solr/core/SolrCore.java	2011-10-19 11:18:43.405730644 +0200
+++ solr-packaging/solr/core/src/java/org/apache/solr/core/SolrCore.java	2011-10-19 11:42:27.830260223 +0200
@@ -369,7 +369,7 @@
 
   void initIndex() {
     try {
-
+      System.setProperty( "org.apache.lucene.lockDir", dataDir );
       initDirectoryFactory();
       String indexDir = getNewIndexDir();
       boolean indexExists = getDirectoryFactory().exists(indexDir);
